src/Tracing/SpanContext.php

```diff
@@ -184,35 +182,4 @@ public function setEndTimestamp(?float $endTimestamp): void
     {
         $this->endTimestamp = $endTimestamp;
     }
-
-    /**
-     * Returns a context populated with the data of the given header.
-     *
-     * @param string $header The sentry-trace header from the request
-     *
-     * @return static
-     */
-    public static function fromTraceparent(string $header)
```

Comment:
This is a breaking change and cannot be done. Please leave the method as-is, mark is as deprecated and throw a `E_DEPRECATED` error following the standard of what has been done in `2.x`